tests/circleci_pipelines/image_referencer/test_manager.py

```diff
@@ -18,3 +18,22 @@ def test_extract_images_from_workflow_no_images(circle_ci_filepath_workflow_no_i
     images = manager.extract_images_from_workflow()
 
     assert not images
+
+def test_extract_images_from_workflow_nested(circle_ci_filepath_workflow_no_images_config):
+    file_path = '/tmp/test_path'
+    config = {
+        'workspace_root': '/go/src/github.com/gruntwork-io/terragrunt',
```

Comment:
can you please add indents so the test case is more readable? 😃

Reply:
Added :)